sdks/java/extensions/sql/zetasql/build.gradle

```diff
@@ -57,6 +64,6 @@ dependencies {
 test {
   dependsOn ":sdks:java:extensions:sql:emptyJar"
   // Pass jars used by Java UDF tests via system properties.
-  systemProperty "beam.sql.udf.test.jar_path", project(":sdks:java:extensions:sql:udf-test-provider").jarPath
-  systemProperty "beam.sql.udf.test.empty_jar_path", project(":sdks:java:extensions:sql").emptyJar.archivePath
+//  systemProperty "beam.sql.udf.test.jar_path", project(":sdks:java:extensions:sql:udf-test-provider").jarPath
```

Comment:
Uncomment these.

Reply:
Oops. These were for getting past a local issue I was having. Didn't mean to push this change. Will revert.